include/sound/sound.h

```diff
@@ -13,13 +13,13 @@
 
 namespace ace { namespace sound {
     // is this cheating
-    using abo = gl::GLObj<alGenBuffers, alDeleteBuffers, ALuint>;
+    using _abo = gl::GLObj<alGenBuffers, alDeleteBuffers, ALuint>;
```

Comment:
Hmm, what's happening here? If GCC isn't liking a variable deceleration with the same name as it's type, I'd prefer the variable be renamed instead.